src/scripts/components/Wallet.js

```diff
@@ -12,7 +11,7 @@ type Props = {
   userAddress: string,
   isWalletSecured: boolean,
   closeModal: () => void,
-  openModal: string => void
+  openModal: () => void
```

Comment:
openModal need the string of the modal you want to open. why do you change w/ : `() => void`